### ISSUE TITLE
Bugs variés

### ISSUE THREAD

**Issue by Angor-de-Redjak** [Zakarik/foundry-knight#135]
- [ ] Le mode Watchtower de la Paladin de divise pas la réaction par deux alors qu'il le devrait

- [ ] Bug d'ordre général. Situation : Il y a un champ numérique associé à une capacité d'armure (style mode boréalis support, taille du mode Goliath) et on tape le chiffre dans la case associée (par exemple 5 pour le nombre d'alliés qu'on veut couvrir avec le mode boréalis support). Il y a alors un petit trait qui indique qu'on tape dedans. Si on clique direct sur le bouton d'activation, la valeur récupérée dans la case est l'ancienne. Pour que ça marche actuellement, il faut cliquer ailleurs que dans la case contenant les chiffres pour que ça soit enregistré, et ensuite on peut activer.
C'est un peu gênant.

**Comment by Zakarik** [comment]
Pour le premier problème, aucun soucis, il va être réglé.

Pour le second problème, ce n'est pas vraiment dépendant de moi.
Ca dépend du navigateur, ou de Foundry lui-même.
Dans mes tests, par exemple, fait à travers le logiciel Foundry, ça fonctionne bien à priori.
Après, selon les cas, il y a peut-être un délai d'enregistrement qui ne se fait pas assez vite, je ne sais pas.

Dans tous les cas, le problème est le même sur Roll20 par exemple, et je ne peux pas y faire grand-chose, hormis rajouter un clique de validation obligatoire avant de pouvoir cliquer sur activer, mais ce n'est pas vraiment une solution non plus.

**Comment by Angor-de-Redjak** [comment]
Effectivement, pour le second je viens de checker sur le système D&D5, même comportement.
Je pense que la raison pour laquelle je ne l'avais jamais remarqué sur D&D c'est qu'il y a peu de moments où on modifie le contenu d'une case pour ensuite cliquer sur un bouton.
Là où par contre dans Knight, c'est bien plus monnaie courante.
À voir mais est-ce que ça ne serait pas possible de rajouter un petit contour coloré tant que la donnée de la case n'a pas été sauvegardée ? Je sais pas si c'est faisable au niveau de l'API. Et c'est sans doute compliqué à mettre en place.

(Si jamais je suis sur Firefox 120.0.1)
